test/ClangImporter/experimental_diagnostics_cmacros.swift

```diff
@@ -34,7 +34,7 @@ _ = INVALID_ARITHMETIC_1
 // CHECK-NEXT: _ = INVALID_ARITHMETIC_1
 // CHECK-NEXT:     ^~~~~~~~~~~~~~~~~~~~
 // CHECK-NEXT:      macros.h:{{[0-9]+}}:9: note: macro 'INVALID_ARITHMETIC_1' not imported: structure not supported
-// CHECK-NEXT:      #define INVALID_ARITHMETIC_1 5 + INVALID_INTERGER_LITERAL_1
+// CHECK-NEXT:      #define INVALID_ARITHMETIC_1 5 + INVALID_INTEGER_LITERAL_1
```

Comment:
This likely needs a corresponding change in `test/Inputs/clang-importer-sdk/usr/include/macros.h` to avoid breaking the test. AFAICT this misspelling was unintentional